p2p/README.md

```diff
@@ -1,6 +1,6 @@
 # P2P subsystem
 
-TODO
+TODO: this document is very incomplete.
 
```

Comment:
Do we need a separate task/issue for this?

Reply:
Sure... why not.